client/core/blockchain.go

```diff
@@ -940,8 +937,8 @@ func writePositionalMetadata(batch kcoindb.Batch, block *types.Block, state *sta
 	rawdb.WritePreimages(batch, block.NumberU64(), state.Preimages())
 }
 
-func IsHead(block *types.Block, currentBlock *types.Block) bool {
-	return block.ParentHash() != currentBlock.Hash()
+func IsParent(currentBlock *types.Block, block *types.Block) bool {
```

Comment:
maybe this would suit better as a receiver on block

IsParent and IsChild

Reply:
Ok. I think IsParent is enough.